### DISCUSSION TITLE
Show active discussion count as same as Issues/Pull Requests

### DISCUSSION THREAD

**Discussion by ryota-murakami** [community/community#48750] [Discussions] [answered]
### Select Topic Area

Product Feedback

### Body

I think that would be better to show discussion count number as same as  Issues/Pull Requests.  
Current design feels like "inactive" for me.

<img width="429" alt="Screenshot 2023-02-28 at 10 29 52" src="https://user-images.githubusercontent.com/5501268/221730891-2f41619e-51b0-4200-99aa-a3b47cead025.png">

**Comment by LindseyB** [comment]
This is coming soon.

**Reply by ryota-murakami** [reply]
@LindseyB Right? Why do you know it?

**Reply by LindseyB** [reply]
I work on the discussions team.

**Reply by ryota-murakami** [reply]
Wow, that's awesome!
Thank you for tell me about it!

**Reply by vladjerca** [reply]
@LindseyB this didn't age well 😛

**Comment by faho** [comment]
I now have this, and I have to say I hate it.

This now turns discussions into yet another "task" to be "done", and makes it annoying to use discussions for long-running concerns - picture a "post your config" thread or similar. This now *feels* like it counts against you because the number *feels* like it should go down.

It also further blurs the already blurry distinction between discussions and "issues".

I would like to disable this because it's incompatible with my brain.

**Reply by LindseyB** [reply]
Understood. We shipped it to experiment, but ultimately it wasn't doing the job we hoped it would of showing how active the community was (in a positive way). It has been permanently disabled now.

**Reply by faho** [reply]
That's great to hear, thanks!

**Comment by annaesvensson** [comment]
I was positively surprised to see the number and how much work our community has done. 

Maybe just maybe, this will invite users to take action and [create more discussions](https://github.com/datenstrom/yellow/discussions/685).

**Comment by ryota-murakami** [comment]
@EricThomson Thanks for comments your insight!
Yeah, it great to show "how active user/maintainer discussion here" visual feedback.

BTW @LindseyB  said "This is coming soon." at Apr 11, 2023 but seems never ship yet?

**Reply by EricThomson** [reply, marked as answer]
@ryota-murakami my understand is that the feature of having the number of discussions in the tab was added and removed already :smile:. I think this was done for reasonable reasons: maintainers didn't like it because it ended up looking like yet anther thing that devs needed to reduce to zero (see comment from @faho above, and the reply from @LindseyB on the feature being disabled). 

My comment here was meant to brainstorm about alternative forms of visual feedback because the current state, of having the tab look the same for all repos, regardless of the state of Discussion (enabled, disabled, etc), seems suboptimal. However, : I didn't realize the tab actually isn't there when Discussions isn't enabled, so I was wrong about that. So it is only the same regardless of the level of activity of discussions, but honestly that's fine. The tab itself is saying "come on over and discuss here".  When Discussions is not enabled, the tab is not there. I think the current situation is good. Sorry for spawning additional confusion over this.

**Reply by ryota-murakami** [reply]
@EricThomson Thanks well explained about the feature!
I think I should close here and waiting for someone open polished alternative idea!

**Reply by vladjerca** [reply]
My takeaway is that this was removed such that people don't suffer from **(self-inflicted)** anxiety?

There is not way to monitor ATM if a new discussion has been started 😕
